pkg/storage/storage.go

```diff
@@ -163,12 +162,7 @@ type RelationshipTupleReader interface {
 	// mandatory ReadPageOptions options. PageSize will always be greater than zero.
 	// It returns a slice of tuples along with a continuation token. This token can be used for retrieving subsequent pages of data.
 	// There is NO guarantee on the order of the tuples in one page.
-	ReadPage(
-		ctx context.Context,
-		store string,
-		tupleKey *openfgav1.TupleKey,
-		options ReadPageOptions,
-	) ([]*openfgav1.Tuple, []byte, error)
+	ReadPage(ctx context.Context, store string, tupleKey *openfgav1.TupleKey, options ReadPageOptions) ([]*openfgav1.Tuple, string, error)
```

Comment:
This is a breaking change for anyone implementing their own storage adapter. Please add an entry in the changelog and call this out.